core/include/kernel/dt.h

```diff
@@ -269,7 +268,7 @@ static inline int _fdt_read_uint32_array(const void *fdt __unused,
 					 int node __unused,
 					 const char *prop_name __unused,
 					 uint32_t *array __unused,
-					 uint32_t count __unused)
+					 uint32_t size_t __unused)
```

Comment:
s/uint32_t size_t/size_t count/
(commit "fixup! core: dt: add kernel DT API to retrieved device information from DT")